src/osx/webrequest_urlsession.mm

```diff
@@ -434,6 +436,13 @@ void Wait()
         return wxString();
 }
 
+std::vector<wxString> wxWebResponseURLSession::GetAllHeaderValues(const wxString& name) const
+{
+    // TODO: URLSession does not support multiple headers with the same name.
```

Comment:
Considering `addValue` description, it would be logical — but I didn't test it at all — if `NSURLSession` also combined multiple header values into a single comma-separated string.